install_scripts/templates/kubernetes/yml-generate.sh

```diff
@@ -887,43 +877,6 @@ spec:
         - mountPath: /etc/ceph
           name: default-config-dir
         env:
-        # To disable RBAC, uncomment the following:
-        # - name: RBAC_ENABLED
-        #  value: "false"
-        # Rook Agent toleration. Will tolerate all taints with all keys.
-        # Choose between NoSchedule, PreferNoSchedule and NoExecute:
-        # - name: AGENT_TOLERATION
-        #   value: "NoSchedule"
-        # (Optional) Rook Agent toleration key. Set this to the key of the taint you want to tolerate
-        # - name: AGENT_TOLERATION_KEY
-        #   value: "<KeyOfTheTaintToTolerate>"
-        # (Optional) Rook Agent mount security mode. Can by `Any` or `Restricted`.
```

Comment:
bash was trying to execute `Any` and `Restricted` so I just deleted the whole comment